pkg/executor/executortype/newdeploy/newdeploymgr.go

```diff
@@ -153,6 +153,14 @@ func (deploy *NewDeploy) DeleteFuncSvcFromCache(fsvc *fscache.FuncSvc) {
 	deploy.fsCache.DeleteEntry(fsvc)
 }
 
+func (deploy *NewDeploy) UnTapService(fn *fv1.Function, svcHost string) {
+	// TO DO
+}
+
+func (deploy *NewDeploy) GetActiveInstances(fn *fv1.Function) int {
+	return 0
+}
+
```

Comment:
Can you please leave a detailed comment :)

Reply:
Done